app/controllers/unattended_controller.rb

```diff
@@ -182,7 +182,7 @@ def set_content_type
 
   def load_template_vars
     # load the os family default variables
-    send "#{@host.os.pxe_type}_attributes"
+    send "#{@host.operatingsystem.pxe_type}_attributes"
```

Comment:
Doing it all in one PR is fine.  I don't really mind which, maybe @eLobato has an opinion?